misk/web/tabs/web-actions/src/containers/SendRequestCollapseContainer.tsx

```diff
@@ -74,66 +73,88 @@ const SendRequestCollapseContainer = (
   props: { action: IWebActionInternal; tag: string } & IState & IDispatchProps
 ) => {
   const { action, tag } = props
+
   // Determine if Send Request form for the Web Action should be open
-  const isOpen = simpleSelect(
-    props.simpleForm,
+  const isOpen = simpleSelectorGet(props.simpleRedux, [
     `${tag}::ButtonSendRequest`,
     "data"
-  )
-  const url = simpleSelect(props.simpleForm, `${tag}::URL`, "data")
+  ])
+  const url = simpleSelectorGet(props.simpleRedux, [`${tag}::URL`, "data"])
+
   // Pre-populate the URL field with the action path pattern on open of request form
   if (isOpen && !url) {
-    props.simpleFormInput(`${tag}::URL`, action.pathPattern)
+    props.simpleMergeData(`${tag}::URL`, action.pathPattern)
   }
   const method: HTTPMethod =
-    simpleSelect(props.simpleForm, `${tag}::Method`, "data") ||
+    simpleSelectorGet(props.simpleRedux, [`${tag}::Method`, "data"]) ||
     action.dispatchMechanism.reverse()[0]
-  const response = simpleSelect(props.simpleNetwork, `${tag}::Response`)
-  const responseData = response.data
-  const whichFormData = simpleSelect(
-    props.simpleForm,
-    `${tag}::RequestBodyFormInputType`,
-    "data",
-    simpleType.boolean
+
+  // Response with fallback to error
+  const response = simpleSelectorGet(
```

Comment:
This is confusing. What's happening here and why?

Reply:
`simpleSelectorGet` follows Lodash Get syntax (https://lodash.com/docs#get) so the third parameter is the fallback value. In this case, if `Response.data` hasn't been set it looks for to see if an error has been set instead.